cmake/partition_manager.cmake

```diff
@@ -73,6 +73,24 @@ if (NOT static_configuration AND CONFIG_PM_IMAGE_NOT_BUILT_FROM_SOURCE)
     place the image correctly in flash.")
 endif()
 
+# From Documentation: By default, the Partition Manager dynamically
+# places the partitions in memory. However, if you have a deployed
+# product that consists of multiple images, where only a subset of
+# the included images can be upgraded through a firmware update
+# mechanism, the upgradable images must be statically configured.
+if (NOT static_configuration AND (
+  (CONFIG_BOOTLOADER_MCUBOOT  AND NOT CONFIG_SECURE_BOOT        ) OR
+  (CONFIG_SECURE_BOOT         AND NOT CONFIG_BOOTLOADER_MCUBOOT ) 
+))
+      message(WARNING "
+        -----------------------------------------------------------------
+        --- WARNING: Using a single bootloader without pm_static.yml. ---
+        --- Static partitioning is required for a single bootloader   ---
```

Comment:
Maybe change the phrasing to something like this, as it is only required when your development is done and you are ready to release the firmware into the wild.
```
        --- WARNING: Using a single bootloader without pm_static.yml. ---
        --- Static partitioning requires using a single bootloader when     ---
        --- releasing firmware. Only use dynamic during development.    ---

Reply:
Good point!
Fixed in https://github.com/nrfconnect/sdk-nrf/commit/16177a0cd38bead290beb466e2a19b2e0f2d2cbc